e2e/capabilities/workflow_test.go

```diff
@@ -0,0 +1,521 @@
+			sc.NewTXOpts(),
+			sc.Client,
+		)
+		require.NoError(t, err)
+		_, err = bind.WaitMined(context.Background(), sc.Client, tx)
+		require.NoError(t, err)
+		fmt.Println("Deployed ocr3_capability contract at", ocr3CapabilityAddress.Hex())
+
+		feedsConsumerAddress, tx, _, err := feeds_consumer.DeployKeystoneFeedsConsumer(
+			sc.NewTXOpts(),
+			sc.Client,
+		)
+		require.NoError(t, err)
+		_, err = bind.WaitMined(context.Background(), sc.Client, tx)
+		require.NoError(t, err)
+		fmt.Println("Deployed feeds_consumer contract at", feedsConsumerAddress.Hex())
+
+		// Add bootstrap spec to the first node
+		bootstrapNode := nodeClients[0]
+
+		var wg sync.WaitGroup
+		wg.Add(1)
+		go func() {
+			defer wg.Done()
+			bootstrapJobSpec := fmt.Sprintf(`
+				type = "bootstrap"
+				schemaVersion = 1
+				name = "Botostrap"
+				contractID = "%s"
+				contractConfigTrackerPollInterval = "1s"
+				contractConfigConfirmations = 1
+				relay = "evm"
+
+				[relayConfig]
+				chainID = %s
+			`, ocr3CapabilityAddress, bc.ChainID)
+			fmt.Println("Creating bootstrap job spec", bootstrapJobSpec)
+			r, _, err2 := bootstrapNode.CreateJobRaw(bootstrapJobSpec)
+			require.NoError(t, err2)
+			require.Equal(t, len(r.Errors), 0)
+			fmt.Printf("Response from bootstrap node: %x\n", r)
+		}()
+
+		ocr3Config := generateOCR3Config(t, nodeClients)
+
+		p2pKeys, err := bootstrapNode.MustReadP2PKeys()
+		require.NoError(t, err)
+		fmt.Println("P2P keys fetched")
+
+		for i, nodeClient := range nodeClients {
+			// First node is a bootstrap node, so we skip it
+			if i == 0 {
+				continue
+			}
+
+			wg.Add(1)
+			go func() {
+				defer wg.Done()
+				scJobSpec := `
+				type = "standardcapabilities"
+				schemaVersion = 1
+				name = "streams-capabilities"
+				command="/streams"`
+				fmt.Println("Creating standard capabilities job spec", scJobSpec)
+				response, _, err2 := nodeClient.CreateJobRaw(scJobSpec)
+				require.NoError(t, err2)
+				require.Equal(t, len(response.Errors), 0)
+				fmt.Printf("Response from node %d after streams SC: %x\n", i+1, response)
+
+				consensusJobSpec := fmt.Sprintf(`
+					type = "offchainreporting2"
+					schemaVersion = 1
+					name = "Keystone OCR3 Consensus Capability"
+					contractID = "%s"
+					ocrKeyBundleID = "%s"
+					p2pv2Bootstrappers = [
+						"%s@%s",
+					]
+					relay = "evm"
+					pluginType = "plugin"
+					transmitterID = "%s"
+
+					[relayConfig]
+					chainID = "%s"
+
+					[pluginConfig]
+					command = "/usr/local/bin/chainlink-ocr3-capability"
+					ocrVersion = 3
+					pluginName = "ocr-capability"
+					providerType = "ocr3-capability"
+					telemetryType = "plugin"
+
+					[onchainSigningStrategy]
+					strategyName = 'multi-chain'
+					[onchainSigningStrategy.config]
+					evm = "%s"
+					`,
+					ocr3CapabilityAddress,
+					nodesInfo[i].OcrKeyBundleID,
+					p2pKeys.Data[0].Attributes.PeerID,
+					strings.TrimPrefix(nodeset.CLNodes[0].Node.HostP2PURL, "http://"),
```

Comment:
should be using Docker URL instead since communication is performed inside Docker network